src/components/gcds-checkbox/gcds-checkbox.css

```diff
@@ -0,0 +1,118 @@
+:host {
+  .gcds-checkbox-wrapper {
+    max-width: 90%;
+    font-family: var(--gcds-font-families-body), sans-serif;
+    font-weight: var(--gcds-font-weights-regular);
+    font-size: var(--gcds-font-sizes-paragraph);
+    line-height: var(--gcds-line-heights-paragraph);
+    color: var(--gcds-checkbox-default-text);
+    margin: 0;
+    padding: 0;
+    border: 0;
+    transition: color ease-in-out 0.15s;
+
+    position: relative;
+    margin-bottom: 20px;
+    margin-top: 28px;
+
+    &:focus-within {
+      color: var(--gcds-checkbox-focus);
+    }
+
+    &.gcds-disabled {
```

Comment:
We are using the class `disabled` in the input and textarea components. Can we stay consistent and either rename this class to `disabled` or also rename the input + textarea classes to `gcds-disabled`?

Reply:
I will create a separate PR for changing the input and textarea component classes to not bloat this PR.